spec/spec_helper.rb

```diff
@@ -18,7 +19,7 @@ module SpecHelper
 RSpec.configure do |config|
   # Set metadata so smoke tests are run on all cop specs
   config.define_derived_metadata(
-    file_path: %r{/spec/rubocop/cop/rspec/(?!mixin/)}
+    file_path: %r{/spec/rubocop/cop/rspec_rails/}
```

Comment:
rubocop-factory_bot [has this line instead](https://github.com/rubocop/rubocop-factory_bot/blob/fe2b31d038c197be211f74ba214879a58d31df80/spec/spec_helper.rb#L21):
```suggestion
    file_path: %r{/spec/rubocop/cop/}
```